spec.md

```diff
@@ -37,7 +37,7 @@ interface Position {
 ```js
 interface Program <: Node {
     type: "Program";
-    body: [ Statement ];
+    body: [Directive, ..., Statement ];
```

Comment:
`[Directive | Statement]` makes sense. 

My concern would be that it does not specify the order of the list. Ideally, I'd like `[Directive..., Statement...]` but I doubt that is supported.